developers/weaviate/config-refs/schema.md

```diff
@@ -42,59 +40,73 @@ Additional configurations are available to help the auto-schema infer properties
 
-### Class
+### Collection
 
-A class describes a data object, such as in the form of a noun (e.g., *Person*, *Product*, *Timezone*) or a verb (e.g., *Move*, *Buy*, *Eat*).
+A collection describes a data object.
 
-Classes are always written with a **capital letter** first. This helps in distinguishing classes from primitive data types when used in properties. For example, `dataType: ["text"]` means that a property is a text, whereas `dataType: ["Text"]` means that a property is a cross-reference type to a class named `Text`.
+Collection names are always written with a **capital letter** first. This helps to distinguish collections from primitive data types when the name is used as a property value. Consider these examples using the `dataType` property:
 
-After the first letter, classes may use any GraphQL-compatible characters. The current (as of `v1.10.0+`) class name validation regex is `/^[A-Z][_0-9A-Za-z]*$/`.
+- `dataType: ["text"]` is a text
+- `dataType: ["Text"]` is a cross-reference type to a collection named `Text`.
 
-:::info Capitalization
-Class and property names are treated equally no matter how the first letter is cased, eg "Article" == "article".
+After the first letter, collection names may use any GraphQL-compatible characters. The collection name validation regex is `/^[A-Z][_0-9A-Za-z]*$/`.
 
-Generally, however, Weaviate follows GraphQL conventions where classes start with a capital letter and properties start with a lowercase letter.
-:::
+import initialCaps from '/_includes/schemas/initial-capitalization.md'
+
+<initialCaps />
 
 ### Properties
 
-Every class has properties. Properties define what kind of data values you will add to an object in Weaviate. In the schema, you define at least the name of the property and its [dataType](../config-refs/datatypes.md). Property names allow `/[_A-Za-z][_0-9A-Za-z]*/` in the name.
+Every collection has properties. Properties define the kinds of data that you add to an object in Weaviate. In the schema, you define at least the name of the property and its [dataType](../config-refs/datatypes.md). Property names contain the following characters: `/[_A-Za-z][_0-9A-Za-z]*/`.
 
-## Class object
+## Collection object
 
-An example of a complete class object including properties:
+An example of a complete collection object including properties:
 
 ```json
 {
-  "class": "Article",                        // The name of the class in string format
-  "description": "An article",                  // A description for your reference
-  "vectorIndexType": "hnsw",                // Defaults to hnsw, can be omitted in schema definition since this is the only available type for now
+  "class": "Article",                       // The name of the collection in string format
```

Comment:
Ln 70-125, mostly reformatting the code comments so they don't run off the screen. Also two replacements of class with collection